pkg/controllers/state/statenode.go

```diff
@@ -185,7 +185,7 @@ func (in *StateNode) ValidateDisruptable(ctx context.Context, kubeClient client.
 	if in.Nominated() {
 		return nil, fmt.Errorf("state node is nominated for a pending pod")
 	}
-	if _, ok := in.Annotations()[v1beta1.DoNotDisruptAnnotationKey]; ok {
```

Comment:
I think we just want to say that this is a presence or lack of presence thing. We do the same thing with `karpenter.sh/do-not-disrupt` where we just check for the presence of the annotation. 

Do we need users to be able to set this value to false? IMO, we just not let that mean anything. Let's update our docs to say that this is a presence/lack-of-presence kind of thing